tests/lib/prepare.sh

```diff
@@ -790,6 +790,34 @@ EOF
                            --output "$IMAGE_HOME/$IMAGE"
     rm -f ./pc-kernel_*.{snap,assert} ./pc_*.{snap,assert} ./snapd_*.{snap,assert}
 
+    if is_core20_system; then
+        # (ab)use ubuntu-seed
+        LOOP_PARTITION=2
+    else
+        LOOP_PARTITION=3
+    fi
+
+    # expand the uc16 and uc18 images a little bit (400M) as it currently will
+    # run out of space easily from local spread runs if there are extra files in
+    # the project not included in the git ignore and spread ignore, etc.
+    if ! is_core20_system; then
+        dd if=/dev/zero bs=1M count=400 >> "$IMAGE_HOME/$IMAGE"
```

Comment:
I think you can use `seek` for the same effect, without actually copying bytes and using storage.

Reply:
In fact, a simple `truncate --size=... fname` line will do. As for `dd` you can use `conv=sparse` to have `dd` seek instead of writing empty blocks.

Reply:
Replaced with `truncate --size=+<size>`.